pkg/cmd/pr/create/create.go

```diff
@@ -536,7 +537,8 @@ func createRun(opts *CreateOptions) error {
 				Repo:      ctx.PRRefs.BaseRepo(),
 				State:     state,
 			}
-			err = shared.MetadataSurvey(opts.Prompter, opts.IO, ctx.PRRefs.BaseRepo(), fetcher, state)
+			// TODO wm: revisit project support
+			err = shared.MetadataSurvey(opts.Prompter, opts.IO, ctx.PRRefs.BaseRepo(), fetcher, state, gh.ProjectsV1Supported)
```

Comment:
This block is untested, however I'm going to rely on the fact it's a simple value pass and the `MetadataSurvey` tests to avoid blowing up this work: https://github.com/cli/cli/pull/10815/files#diff-e76ab3c0932c8cf32002e9ad1bd0c61a3fd6f26d9bdcbbc65fdb30ec14e41a41R133-R175